pkg/client/client_test.go

```diff
@@ -234,7 +234,7 @@ func testObjects(t *testing.T) {
 
 	assert.Equal(t, nil, err)
 	assert.Equal(t, []string{"schema", "name", "type", "owner", "comment"}, res.Columns)
-	assert.Equal(t, []string{"public"}, mapKeys(objects))
+	assert.Equal(t, []string{"pg_temp_1", "public"}, mapKeys(objects))
```

Comment:
Perhaps it'd be better to ignore all temp schemes. `make test-all` worked that way.